_data/releases.tsv

```diff
@@ -53,3 +53,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2021-05-31	prod97	0.9.7	17466	1793693		11817796	86892269	266.539897252345	29.565929	554.51
 2021-06-30	prod98	0.9.8	18489	1793693		12313512	88620001	279.869801734009	30.066415	581.5
 2021-08-03	prod99	0.9.9	18513	1793693		12319306	95017392	280.876962901964	30.08391	588.76
+2021-08-31	prod100	0.10.0	18557	1795095		12335276	98870307	283.3208304418	30.36849	539.68
```

Comment:
With https://github.com/IDR/idr-utils/pull/45 I get the same metrics with the sole exception being the total size where I get `282.698012588988` instead of `283.3208304418`. Some rounding error due to the size of `idr0117`?

Reply:
282.69 is correct. I summed up the "Size" column instead of the "Size (TB)" column earlier. Somewhere there must be a rounding error in there.